src/EntityWithOdmInterface.php

```diff
@@ -0,0 +1,10 @@
+<?php
+declare(strict_types=1);
+
+namespace Goodwix\DoctrineJsonOdm;
+
+interface EntityWithOdmInterface
```

Comment:
Hello, @mkSpitfire ! What's the purpose of this feature? Can you write any tests?